hatchet_sdk/loader.py

```diff
@@ -127,6 +136,28 @@ def get_config_value(key, env_var):
 
         tls_config = self._load_tls_config(config_data["tls"], host_port)
 
+        otel_exporter_oltp_endpoint = get_config_value(
+            "otel_exporter_oltp_endpoint", "HATCHET_CLIENT_OTEL_EXPORTER_OTLP_ENDPOINT"
+        )
+
+        otel_service_name = get_config_value(
+            "otel_service_name", "HATCHET_CLIENT_OTEL_SERVICE_NAME"
+        )
+
+        _oltp_headers = get_config_value(
+            "otel_exporter_oltp_headers", "HATCHET_CLIENT_OTEL_EXPORTER_OTLP_HEADERS"
+        )
+
+        try:
+            otel_exporter_oltp_headers = json.loads(_oltp_headers) if isinstance(_oltp_headers, str) else _oltp_headers
+        except json.JSONDecodeError:
+            otel_exporter_oltp_headers = None
```

Comment:
this is a hack - I'm sure there's a nicer way to do this (basically need to get the headers in some structured format and parse them to a dict)